engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionNode.java

```diff
@@ -74,13 +72,15 @@ public AmbientOcclusionNode(Context context) {
         addDesiredStateChange(new EnableMaterial(SSAO_MATERIAL));
         ssaoMaterial = getMaterial(SSAO_MATERIAL);
 
-        displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
-        requiresFBO(new FBOConfig(SSAO_FBO, FULL_SCALE, FBO.Type.DEFAULT), displayResolutionDependentFBOs);
-        addDesiredStateChange(new BindFbo(SSAO_FBO, displayResolutionDependentFBOs));
-        addDesiredStateChange(new SetViewportToSizeOf(SSAO_FBO, displayResolutionDependentFBOs));
-        update(); // Cheeky way to initialise ssaoFbo, outputFboWidth, outputFboHeight
+        DisplayResolutionDependentFBOs displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
+        ssaoFbo = requiresFBO(new FBOConfig(SSAO_FBO, FULL_SCALE, FBO.Type.DEFAULT), displayResolutionDependentFBOs);
+        addDesiredStateChange(new BindFbo(ssaoFbo));
+        addDesiredStateChange(new SetViewportToSizeOf(ssaoFbo));
+        update(); // Cheeky way to initialise outputFboWidth, outputFboHeight
         displayResolutionDependentFBOs.subscribe(this);
 
+        ShaderParametersSSAO.setGBufferRead(displayResolutionDependentFBOs.getGBuffer().getWriteFbo());
+
```

Comment:
I can't seem to find any indentation issue here (neither on github, nor in IntelliJ). That being said, it did look weird on my mobile in GMail.. strange.